Modules/Components/itkImageRegistrationMethodv4/include/selxItkImageRegistrationMethodv4Component.hxx

```diff
@@ -122,6 +122,7 @@ InternalComputationValueType >::ItkImageRegistrationMethodv4Component( const std
 {
   this->m_ImageRegistrationMethodv4Filter = ImageRegistrationMethodv4Type::New();
   this->m_InvertIntensity = false;
+  this->m_MetricSamplingPercentage = 1.0;
```

Comment:
Looks fine, although please note that this is not the modern C++ way to initialize a data member. In general, it is recommended to either use the member-initializer-list of the constructor, or an in-class default-member-initializer (introduced with C++11). With ITK5, lots of code is adapted to start using modern C++11 in-class default-member-initializers: https://discourse.itk.org/t/simpletype-m-var-initialvalue/651/8

Reply:
Yes, will change